### PR TITLE
Vue routes

### PR DESCRIPTION
**Pull request by felixsand**

### CODE REVIEW

File: resources/assets/js/views/EnterPassword.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        data () {
+            return {
+                passwordField: '',
+                passwordConfirmField: '',
+                views: 1,
+                hours: 0,
+                days: 1,
+            }
+        },
+        mounted: function () {
```
Comment (Karagath) [inline resources/assets/js/views/EnterPassword.vue:59]: Can be shortened to `mounted: {...}`

File: resources/assets/js/views/EnterPassword.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            this.$emit('error', '');
+        },
+        methods: {
+            viewsChanged: function(views) {
```
Comment (Karagath) [inline resources/assets/js/views/EnterPassword.vue:63]: Can be shortened to `viewsChanged(views) {...}`

File: resources/assets/js/views/EnterPassword.vue [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            <a class="btn btn-primary btn-lg" @click="storeSecret" id="get-url-btn" href="#" role="button">
+                Get URL
+            </a>
+        </p>
+    </div>
+</template>
+
+
+<script>
+    import NofViewsSelector from "../components/NofViewsSelector.vue";
```
Comment (Karagath) [inline resources/assets/js/views/EnterPassword.vue:40]: I think this can be shortened to `from "../components/NofViewsSelector";` but I may be wrong

**Review by Karagath — COMMENTED** [review]
If you want the project more ES 6 :)

**Comment by felixsand** [comment]
Thanks @Karagath ! 🙏 
I'll make the suggested changes 🙂

**Pull request merged**